recipes/libxlsxwriter/all/patches/001-patch-cmake-conan-targets.patch

```diff
@@ -0,0 +1,33 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index 2359933..955563c 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -180,16 +180,16 @@ endif()
+ # INCLUDES
+ # --------
+ enable_language(CXX)
+-list(APPEND CMAKE_MODULE_PATH ${CMAKE_CURRENT_SOURCE_DIR}/cmake)
++#list(APPEND CMAKE_MODULE_PATH ${CMAKE_CURRENT_SOURCE_DIR}/cmake)
+ 
+ # ZLIB
+-find_package(ZLIB REQUIRED "1.0")
++#find_package(ZLIB REQUIRED "1.0")
+ list(APPEND LXW_PRIVATE_INCLUDE_DIRS ${ZLIB_INCLUDE_DIRS})
+ message("zlib version: " ${ZLIB_VERSION})
+ 
+ # MINIZIP
+ if (USE_SYSTEM_MINIZIP)
+-    find_package(MINIZIP REQUIRED "1.0")
++    #find_package(MINIZIP REQUIRED "1.0")
+     list(APPEND LXW_PRIVATE_INCLUDE_DIRS ${MINIZIP_INCLUDE_DIRS})
```

Comment:
If conan would have been able to modify the behavior of the cmake_find_package generator inside consumers, this patch would not have been necessary.